.github/workflows/build-service.yml

```diff
@@ -35,7 +35,7 @@ env:
   # relatively stable Pipelines containers that are needed to avoid issues with
   # the "latest" version; they would remain in this list until "latest" becomes
   # usable for all building and testing.
-  BASE_TAG_LIST: '["latest"]'
+  BASE_TAG_LIST: '["DM-47975-redis-d_2025_01_31"]'
```

Comment:
I recommend using `DM-47975-redis-latest`, since that's what the PR base build [is actually building](https://github.com/lsst-dm/prompt_processing/pkgs/container/prompt-base) (though, since the builds are triggered simultaneously, any changes to the base container will still be one iteration out of date).